ld-relay.go

```diff
@@ -47,6 +49,7 @@ type Config struct {
 	Redis struct {
 		Host string
 		Port int
+		Ttl  *int
```

Comment:
but if it is consistent with how it is named in the RedisFeatureStore we should either leave it or change it in both places